src/main/java/com/gmail/goosius/siegewar/command/SiegeWarAdminCommand.java

```diff
@@ -102,42 +133,58 @@ private void parseSiegeWarAdminCommand(CommandSender sender, String[] args) {
 			showHelp(sender);
 		}
 	}
-	
+
 	private void showHelp(CommandSender sender) {
 		sender.sendMessage(ChatTools.formatTitle("/siegewaradmin"));
 		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "reload", Translation.of("admin_help_1")));
 		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "immunity town [town_name] [hours]", ""));
 		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "immunity nation [nation_name] [hours]", ""));
-		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "immunity all towns [hours]", ""));
-	
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "immunity alltowns [hours]", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "siege [town_name] setpoints [points]", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "siege [town_name] end", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "town [town_name] setplundered [true/false]", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "town [town_name] setcaptured [true/false]", ""));
 	}
-	
+
 	private void showImmunityHelp(CommandSender sender) {
 		sender.sendMessage(ChatTools.formatTitle("/swa immunity"));
-		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa immunity", "town [town_name] [hours]", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "immunity town [town_name] [hours]", ""));
 		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "immunity nation [nation_name] [hours]", ""));
-		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa immunity", "all towns [hours]", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "immunity alltowns [hours]", ""));
+	}
+
+	private void showSiegeHelp(CommandSender sender) {
+		sender.sendMessage(ChatTools.formatTitle("/swa siege"));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "siege [town_name] setpoints [points]", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "siege [town_name] end", ""));
+	}
 
+	private void showTownHelp(CommandSender sender) {
+		sender.sendMessage(ChatTools.formatTitle("/swa town"));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "town [town_name] setplundered [true/false]", ""));
+		sender.sendMessage(ChatTools.formatCommand("Eg", "/swa", "town [town_name] setcaptured [true/false]", ""));
 	}
 
 	private void parseSiegeWarReloadCommand(CommandSender sender) {
 		if (Settings.loadSettingsAndLang()) {
 			Messaging.sendMsg(sender, Translation.of("config_and_lang_file_reloaded_successfully"));
 			return;
 		}
-		
+
 		Messaging.sendErrorMsg(sender, Translation.of("config_and_lang_file_could_not_be_loaded"));
 	}
 
 	private void parseSiegeWarImmunityCommand(CommandSender sender, String[] args) {

```

Comment:
All changes to parseSiegeWarImmunityCommand can be undone except: lines 215,216.:

```java
		} else if(args.length == 2
			&& args[0].equalsIgnoreCase("alltowns")) {
```

Reply:
The parseLong would throw either a NumberFormatException or ArrayIndexOutofbounds exception, since args[1] is the desired hours for alltowns